[v0.50.04-SNAPSHOT] Prisoner defection event failed to list names, missing resources error.

### Brief Description *

After taking a number of prisoners, a defection event pop up happened. 

![Image](https://github.com/user-attachments/assets/d920885f-9a59-42c9-ad4f-80cdc8bb6188)

This pop-up failed to correctly display the name and origin faction of the defecting prisoners. 
When clicking 'Understood', there was a further error: 

![Image](https://github.com/user-attachments/assets/062a54b1-b99f-40ac-94bd-ea18a2e05c80)

In addition, none of the captives actually appears to be willing to defect, despite the notice. 
Assumedly, the system tried to defect an absent / non-existent prisoner.

The log spat out the following: 


```
15:00:09,415 ERROR [mekhq.MekHQ] {AWT-EventQueue-0}
mekhq.MekHQ.lambda$main$1(MekHQ.java:302) - Uncaught java.util.MissingResourceException detected. Please open up an issue containing all logs, the game save file, and customs at https://github.com/MegaMek/megamek/issues . If Sentry is enabled, the error has also been logged.
java.util.MissingResourceException: Can't find bundle for base name mekhq.resources.DefectionOffer, locale en
	at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2059)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1697)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1600)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1555)
	at java.base/java.util.ResourceBundle.getBundle(ResourceBundle.java:1066)
	at mekhq.utilities.MHQInternationalization.lambda$getResourceBundle$0(MHQInternationalization.java:73)
	at java.base/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1708)
	at mekhq.utilities.MHQInternationalization.getResourceBundle(MHQInternationalization.java:72)
	at mekhq.utilities.MHQInternationalization.getTextAt(MHQInternationalization.java:83)
	at mekhq.utilities.MHQInternationalization.getFormattedTextAt(MHQInternationalization.java:116)
	at mekhq.campaign.randomEvents.prisoners.CapturePrisoners.handlePostCapture(CapturePrisoners.java:413)
	at mekhq.campaign.randomEvents.prisoners.CapturePrisoners.processCaptureOfNPC(CapturePrisoners.java:222)
	at mekhq.campaign.ResolveScenarioTracker.resolveScenario(ResolveScenarioTracker.java:1570)
	at mekhq.gui.dialog.ResolveScenarioWizardDialog.finish(ResolveScenarioWizardDialog.java:1557)
	at mekhq.gui.dialog.ResolveScenarioWizardDialog.lambda$initComponents$4(ResolveScenarioWizardDialog.java:314)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2314)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:407)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:279)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6621)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3398)
	at java.desktop/java.awt.Component.processEvent(Component.java:6386)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4996)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4828)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4948)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4575)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4516)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2780)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4828)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:775)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:98)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:747)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:744)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:117)
	at java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:191)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:236)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:234)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:319)
	at java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:234)
	at java.desktop/java.awt.Dialog.show(Dialog.java:1079)
	at java.desktop/java.awt.Component.show(Component.java:1728)
	at java.desktop/java.awt.Component.setVisible(Component.java:1675)
	at java.desktop/java.awt.Window.setVisible(Window.java:1036)
	at java.desktop/java.awt.Dialog.setVisible(Dialog.java:1015)
	at mekhq.MekHQ.gameVictory(MekHQ.java:558)
	at megamek.common.event.GameVictoryEvent.fireEvent(GameVictoryEvent.java:65)
	at megamek.common.AbstractGame.fireGameEvent(AbstractGame.java:196)
	at megamek.common.Game.processGameEvent(Game.java:759)
	at megamek.client.Client.handleGameSpecificPacket(Client.java:1103)
	at megamek.client.AbstractClient.handlePacket(AbstractClient.java:394)
	at megamek.client.AbstractClient$1.lambda$packetReceived$0(AbstractClient.java:566)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: java.lang.NullPointerException
	at java.base/java.io.Reader.<init>(Reader.java:168)
	at java.base/java.io.InputStreamReader.<init>(InputStreamReader.java:123)
	at mekhq.utilities.MHQInternationalization$UTF8Control.newBundle(MHQInternationalization.java:65)
	at java.base/java.util.ResourceBundle.loadBundle(ResourceBundle.java:1998)
	at java.base/java.util.ResourceBundle.findBundle(ResourceBundle.java:1784)
	at java.base/java.util.ResourceBundle.findBundle(ResourceBundle.java:1736)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1670)
	... 75 more

```

### 3. Steps to Reproduce

_No response_

### Attach Files

[mekhq.log](https://github.com/user-attachments/files/19054551/mekhq.log)
[The Hedge Knights-Prisoner defection error.zip](https://github.com/user-attachments/files/19054557/The.Hedge.Knights-Prisoner.defection.error.zip)

### Severity *

Low (Minor/Nuisance): Minor glitches or cosmetic issues that don't affect gameplay and occur rarely.

### MekHQ Suite Version *

v0.50.04-SNAPSHOT

### Operating System *

Windows 11 10.0 (amd64)

### Java Version *

    Java Vendor: Eclipse Adoptium     Java Version: 21.0.3

### Final Verification

- [x] I confirm this is a single, unique issue that hasn't been reported before
- [x] I have filled and provided all necessary information above
- [x] I have included any and all logs, custom units, and CPNX (save) files
- [x] I have asked on MegaMek Discord about this issue
- [x] I have confirmed this issue is being opened on the correct repository: MegaMek, MegaMekLab, or MekHQ

--- a/MekHQ/src/mekhq/campaign/randomEvents/prisoners/CapturePrisoners.java
+++ b/MekHQ/src/mekhq/campaign/randomEvents/prisoners/CapturePrisoners.java
@@ -398,7 +398,7 @@ public class CapturePrisoners {
      * @param newStatus  The resulting {@link PrisonerStatus} of the prisoner post-capture.
      */
     private void handlePostCapture(Person prisoner, PrisonerStatus newStatus) {
-        final String RESOURCE_BUNDLE = "mekhq.resources.DefectionOffer";
+        final String RESOURCE_BUNDLE = "mekhq.resources.PrisonerEvents";
 
         // non-clan prisoners should generate with lower than average loyalty, so drop the highest roll
         if (!newStatus.isBecomingBondsman()) {

